code/ui/ui_playermodel.c

```diff
@@ -1140,8 +1178,13 @@ static void PlayerModel_BuildList(void)
 
     ///Com_Printf("%i folders found\n", numdirs );
 
+    int pool_idx = 0;
+    ht_element_t  ele_pool[MAX_PLAYERCHARS];
```

Comment:
i would prefer some more descriptive variable names. We're not living in the 80's anymore where with 640x480 horizontal screen space came at a premium :p